### PR TITLE
finetuning colab

### PR DESCRIPTION
**Pull request by kldarek**

### CODE REVIEW

File: llm-training-course/colab/finetuning.ipynb [hunk 1]
```diff
@@ -0,0 +1,1100 @@
+{
```
Comment (morganmcg1) [inline llm-training-course/colab/finetuning.ipynb:1]: <p>Move title to above the installs,</p><p> </p><p>Link to W&amp;B  HF documentation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/edu/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (morganmcg1) [inline llm-training-course/colab/finetuning.ipynb:1]: <p>we don't have to pass <code>report_to = "wandb"</code> if using the callback? Maybe show it anyways to get people in the habit of it anyways when using HF Trainer.</p><p> </p><p>Link to W&amp;B HF documentation above this cell too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/edu/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (morganmcg1) [inline llm-training-course/colab/finetuning.ipynb:1]: <p>Link to W&amp;B HF documentation above this cell too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/edu/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (kldarek) [reply llm-training-course/colab/finetuning.ipynb:1]: <p>updated</p>

Comment (kldarek) [reply llm-training-course/colab/finetuning.ipynb:1]: <p>updated</p>

Comment (kldarek) [reply llm-training-course/colab/finetuning.ipynb:1]: <p>updated</p>

**Pull request merged**